rtl_433/run.sh

```diff
@@ -18,8 +18,8 @@ fi
 
 # Check the output options specified in the configuration
 if output_options=$(bashio::config "websocket"); then
-    host=$(bashio::config "ws_http_host")
-    port=$(bashio::config "ws_http_port")
+    host="0.0.0.0"
+    port="8433"
```

Comment:
since this is exclusively for the integration, i don't think there's a reason to ever change the port

Reply:
fair enough, could always be done if someone wanted